keras/models.py

```diff
@@ -1037,6 +1037,7 @@ def fit_generator(self, generator,
                       max_queue_size=10,
                       workers=1,
                       use_multiprocessing=False,
+                      shuffle=False,
```

Comment:
FYI the default in [the other `fit_generator` implementation](https://github.com/fchollet/keras/blob/8ac788a5614570d222c484ab49cf9e878eeab9ff/keras/engine/training.py#L1835) is `shuffle=True`...